docs/notes/instrumentation_measurement/evmone.md

```diff
@@ -3,13 +3,35 @@
 
 ### Installation and running
 
-1. I got compile errors because of old gcc not supporting C++17
+1. Building
```

Comment:
pls leave comment on how to setup evmc and evmone properly - which branches etc.

Reply:
ah got it - `submodule` handles it on both. Just let's note, that both repos should use `wallclock` branch (or any other, where our changes are)